sale_coupon_advanced/views/sale_coupon.program.xml

```diff
@@ -11,7 +11,7 @@
                 <field name="is_cumulative" />
                 <field name="first_order_only" />
                 <field
-                    name="first_n_customer_orders"
+                    name="next_n_customer_orders"
                     attrs="{'invisible': [('first_order_only', '=', True)]}"
```

Comment:
```suggestion
```
now this is unrelated features

Reply:
Technically those two can work together, but does it make sense from business perspective?

Reply:
well yeah they meaningless together, than you need
` attrs="{'invisible': [('first_order_only', '=', False)]}"`
to hide one when the first is raised

Reply:
Well if you mean to put this on `first_order_only`, it would hide it permanently. Something like this should work: 
`attrs="{'invisible': [('first_order_only', '=', True)]}" on next_n_customer_orders`
`attrs="{'invisible': [('next_n_customer_orders', '=', True)]}" on first_order_only`